### DISCUSSION TITLE
change decTalk license to gnu public license

### DISCUSSION THREAD

**Discussion by joshknnd1982** [dectalk/dectalk#34] [General] [answered]
I noticed there are complaints that the decTalk license is not a gnu general public license. Can you please remove the original license and put decTalk under the gnu general public license? thanks.

**Comment by 7coil** [comment]
Licencing is always a strange issue; we don't hold the authority to change the licence of the code without the permission of the original authors of the code.

As a result, I don't think changing the licence is the correct course of action.

However, software built on top of DECtalk that we may build in the future could be licenced with more permissive licences such as the MIT licence.

I'll leave this open for a bit longer to garner some response from the other Devs, but I suspect they'll have a similar response.

Thanks

N.B. Do not doxx, especially the owners of Fonix, DECtalk.

**Comment by joshknnd1982** [comment]
What if I could get an official recorded statement from one of the authors of the code? Would that be acceptable to all of you?

**Reply by DidierSpaier** [reply]
> What if I could get an official recorded statement from one of the authors of the code? Would that be acceptable to all of you? 

I think that would not solve the issue, if this author does not represent the license's issuer.

**Comment by TapGhoul** [comment]
If this is relicensed, I second a non-GPL license. MIT or Apache 2.0 would be far more preferable.

Just keep in mind, the current DECTalk rights holders are extremely litigious - they've been know to copyright strike singing MBAlpha YouTube videos because they didn't own a personal license and they saw it as using it for broadcast, despite the game itself holding one.

**Comment by DidierSpaier** [comment]
Currently the LICENSE document states:

"Possession, use, or copying of this software and media is authorized only pursuant to a valid written license from FONIX or an authorized sublicensor."

 Why not just consult with Fonix? Maybe they could accept to modify the terms of this license making the possession, usage and copying of the software allowed to anyone. This does not need to be the GPL, any other open source listed in https://opensource.org/licenses/ would be acceptable, I think.

As a distribution l maintainer I won't provide a package for this software with the current license.

**Comment by joshknnd1982** [comment]
You’re correct it would not. If I made some software assuming I were a developer my license would go something like this…

As long as I’m alive and I am making money from my software and company you can only buy and use the software on your machine and others have to buy it so I can make money. If at any point my company goes out of business and attempts to contact former employees fail, and attempts to contact former CEO’s fail, then do whatever you want with the software because I no longer care. License agreements should be short and simple, not this big long conveluded mess of legal jargon the average human can’t understand and is so long nobody wants to read it. 

There. If I had software that would be my license agreement.

**Comment by willjr880** [comment, marked as answer]
* Changing the license is not a decision for us to make, nor is it yours to chase up on our behalf. 

* Do not attempt to make any profit from any of this code.

* We highly frown upon anyone harassing people who may have previously had control over the source code or licensing and ask that you stop immediately.

**Comment by DidierSpaier** [comment]
According to this article from Wikipedia:  https://en.wikipedia.org/wiki/SpeechFX Fonix became SpeechFX, however the web site this article links to: http://www.speechfxinc.com/ doesn't seem to exist. So, I am puzzled about whom to speak to...

**Comment by joshknnd1982** [comment]
Don’t you understand these companies are dead and abandoned?

**Comment by joshknnd1982** [comment]
and chasing down the license holders is a waste of time because the phone numbers and emails for them are dead. dectalk is abandoned. abandoned by the people and companies that made it years ago.

**Comment by joshknnd1982** [comment]
do not attempt to profit from the code... Why does everyone's minds immediately jump to profet profet profet, money money money? There's more to life than profet and money. like enjoying and using stuff for free just because its fun to use in order to play old audio games with and to use with screen readers for a classic retro tts sound. And if we want to use it on linux or android or IOS , ok then have at it, have fun. just keep it free and don't sell it. Also in order for me to understand the apache and other licenses I had to ask chat gpt to summarise it for a first grader in simple plain english. No wonder why people don't read license agreements. They are so conveluded its not even funny.

**Comment by willjr880** [comment]
@joshknnd1982 you have asserted your theories and desires across many discussion forums on this matter for quite some time now. Your personal pursuits have only ever resulted in upsetting people. We would like to move on from this discussion once and for all, along with your suggestions about what platform/operating system this project should be ported to next. It is now to the point where it is blatant harassment. Do as you wish with dectalk to have fun, but stop now with your demands and confrontation.
